maven-resolver-provider/src/main/java/org/apache/maven/repository/internal/DefaultArtifactDescriptorReader.java

```diff
@@ -283,16 +283,12 @@ private Model loadPom( RepositorySystemSession session, ArtifactDescriptorReques
         }
     }
 
-    private Properties toProperties( Map<String, String> dominant, Map<String, String> recessive )
+    private Properties toProperties( Map<String, String> map )
     {
         Properties props = new Properties();
-        if ( recessive != null )
+        if ( map != null )
```

Comment:
This check is redudant because the contract of `RepositorySystemSession` says that the return value is never null.